src/components/Home/styles.module.scss

```
@@ -4,9 +4,82 @@
 }
 .hr {
     display: block;
-    margin-top: 5px;
-    margin-bottom: 5px;
+    margin-top: 8px;
+    margin-bottom: 8px;
     background-color: gray;
     opacity: 0.3;
     height: 2px;
-}
\ No newline at end of file
+}
+
+.cont1 {
+    width:100%;
+    height: 70%;
+    display: flex;
+    align-items: flex-end;
+}
+
+.containerHome {
+    width: 40%;
+    font-size: 30px;
+}
+
+.title {
+    color: blue;
+    font-size: 10vh;
+}
+
+.cont2 {
+    width: 100%;
+    height: 30%;
+}
+
+.containerButton {
+    height: auto;
+    width: auto;
+    margin-top: calc(2% + 28px);
+}
+
+
+.button1 {
+    background-color:#FF0080;
+    color: white;
+    font-size: 20px;
+    border-radius: 5px;
+    margin-right: 25px;
+    border-color: #FF0080;
+    width: 250px;
+    height: 64px;
+    transition: all 1.2s;
+    &:hover {
+        transform: translateY(-5px);
```

Es mejor poner espaciados cuando hacemos nesting en SASS para que el código sea más legible.
```suggestion
    transition: all 1.2s;

    &:hover {
        transform: translateY(-5px);
```
También revisaría un poco la duración de la transición, quizá para ser la transición de hover de un botón, sea muy lenta. Te dejo un [artículo](https://material.io/design/motion/speed.html#) donde explica un poco cómo manejar animaciones y transiciones de acuerdo a la manera en que lx usuarix está interactuando con los elementos.

---

Gracias Kike por el articulo, muy bueno !